src/fable/Fable.Compiler/Replacements.fs

```diff
@@ -158,7 +158,12 @@ module Util =
         let patternFor = function
             | Int8 -> "($0 + 0x80 & 0xFF) - 0x80"
             | Int16 -> "($0 + 0x8000 & 0xFFFF) - 0x8000"
-            | Int32 -> "($0 + 0x80000000 >>> 0) - 0x80000000"
+            | Int32 when i.ownerFullName = "System.Convert" -> 
+                // TODO: When the fable round matches .net replace the following:
+                let x = (com : ICompiler).GetUniqueVar()
+                let r = com.GetUniqueVar()
+                "(("+x+") => { var "+r+" = Math.round("+x+"); return (((("+x+">0)?"+x+":(-"+x+"))%1)===0.5)?((0===("+r+"%2))?"+r+":("+r+"-1)):"+r+"; } )($0)"
```

Comment:
My hope/guess is no one in the F# world will be using the `Convert.ToInt32`, this is just a fallback? I'm more than happy to make this a function if you want but I'll let you decided.